### PR TITLE
JBIDE-28449: Extensions from platform should have an indicator

### PR DESCRIPTION
**Pull request by jeffmaury**
Signed-off-by: Jeff MAURY <jmaury@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

File: plugins/org.jboss.tools.quarkus.ui/src/org/jboss/tools/quarkus/ui/QuarkusUIPlugin.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public static void logException(String errMsg, Throwable ex) {
 
 	@Override
 	protected void initializeImageRegistry(ImageRegistry registry) {
-		registry.put(CODESTARTS_EXTENSION_ICON_KEY, ImageDescriptor.createFromFile(QuarkusUIPlugin.class,
-			      "/icon/fighter-jet-solid.png"));
```
Comment (sbouchet) [inline plugins/org.jboss.tools.quarkus.ui/src/org/jboss/tools/quarkus/ui/QuarkusUIPlugin.java:77]: icon is no more needed, please remove it from /icon folder as well

**Review by sbouchet — CHANGES_REQUESTED** [review]

**Comment by sbouchet** [comment]
also, is there any example/extension without the icon yet to test ?

**Comment by jeffmaury** [comment]
> also, is there any example/extension without the icon yet to test ?

Quinoa

**Review by sbouchet — COMMENTED** [review]
LGTM, however the icon is still there ?

**Comment by jeffmaury** [comment]
> LGTM, however the icon is still there ?

It should be a different icon. Previously the icon identified extensions that add sample code, now the icon identifies extensions that are part of the platform

**Comment by sbouchet** [comment]
> > LGTM, however the icon is still there ?
> 
> It should be a different icon. Previously the icon identified extensions that add sample code, now the icon identifies extensions that are part of the platform

yes, i see the new icon in the UI, my remark is more that the oldone is still in the source code repo and should be deleted since it is not used anymore :)

**Comment by jeffmaury** [comment]
> > > LGTM, however the icon is still there ?
> > 
> > 
> > It should be a different icon. Previously the icon identified extensions that add sample code, now the icon identifies extensions that are part of the platform
> 
> yes, i see the new icon in the UI, my remark is more that the old one is still in the source code repo and should be deleted since it is not used anymore :)

Sorry I pushed my commit to another PR. Fixed now

**Review by sbouchet — APPROVED** [review]
LGTM

**Pull request merged**